controllers/metrics.go

```diff
@@ -25,17 +25,25 @@ type DRPolicySyncMetrics struct {
 	DRPolicySyncInterval prometheus.Gauge
 }
 
+const (
+	ObjType               = "obj_type"
+	ObjName               = "obj_name"
+	ObjNamespace          = "obj_namespace"
+	ObjPolicyname         = "obj_policyname"
```

Comment:
We should drop obj from policy name and scheduling interval, other places are fine

Reply:
updated